swagger-codegen/2.1.6.wso2v1/pom.xml

```diff
@@ -0,0 +1,155 @@
+                            JavaJaxRS.*;version="${export.pkg.version.swagger-codegen}",
+                            JavaInflector.*;version="${export.pkg.version.swagger-codegen}",
+                            htmlDocs.*;version="${export.pkg.version.swagger-codegen}",
+                            Groovy.*;version="${export.pkg.version.swagger-codegen}",
+                            flaskConnexion.*;version="${export.pkg.version.swagger-codegen}",
+                            flash.*;version="${export.pkg.version.swagger-codegen}",
+                            dart.*;version="${export.pkg.version.swagger-codegen}",
+                            CsharpDotNet2.*;version="${export.pkg.version.swagger-codegen}",
+                            csharp.*;version="${export.pkg.version.swagger-codegen}",
+                            config.*;version="${export.pkg.version.swagger-codegen}",
+                            codegen.*;version="${export.pkg.version.swagger-codegen}",
+                            clojure.*;version="${export.pkg.version.swagger-codegen}",
+                            asyncscala.*;version="${export.pkg.version.swagger-codegen}",
+                            android.*;version="${export.pkg.version.swagger-codegen}",
+                            akka-scala.*;version="${export.pkg.version.swagger-codegen}",
+                            Java.*;version="${export.pkg.version.swagger-codegen}",
+                            io.swagger.*;version="${export.pkg.version.swagger-codegen}"
+                        </Export-Package>
+                        <Import-Package>
+                            !io.swagger.codegen.*,
+                            !io.swagger.parser.*,
+                            org.apache.commons.io;version="1.4",
+                            org.apache.commons.io.comparator;version="1.4",
+                            org.apache.commons.io.filefilter;version="1.4",
+                            org.apache.commons.io.input;version="1.4",
+                            org.apache.commons.io.output;version="1.4",
+                            org.joda.time.*;version="2.9.2",
+                            io.swagger.models.auth; version="1.5.8",
+                            io.swagger.models.parameters; version="1.5.8",
+                            io.swagger.models.properties; version="1.5.8",
+                            io.swagger.models.refs; version="1.5.8",
+                            io.swagger.models; version="1.5.8"
+                        </Import-Package>
+                        <Private-Package>
+                        </Private-Package>
+                        <Include-Resource>
+                            {maven-resources}
+                        </Include-Resource>
+                        <Embed-Dependency>
+                            joda-time;scope=compile|runtime;inline=false,
+                            commons-io;scope=compile|runtime;inline=false
+                        </Embed-Dependency>
```

Comment:
Some packages from joda-time and commons-io are needed for swagger-codegen but they are not already bundles so instead of creating bundles for them I have added them as embed dependencies.